global.json

```diff
@@ -1,3 +1,4 @@
 {
+    "packages": "packages",
```

Comment:
Ignore that, I'll remove it when we're done with the main changes (I didn't want the CI dev packages to be added to my global packages folder).